jsonpointer.js

```diff
@@ -1,5 +1,3 @@
-var console = require("console");
```

Comment:
I still don't know whether this is here to support IE6&IE7.
In each other scenario we can remove that.